pages/modeling.js

```diff
@@ -3,10 +3,10 @@ import { Grid } from "@material-ui/core";
 import AddIcon from '@mui/icons-material/Add';
 import Button from '@mui/material/Button';
 import ArrowBackIcon from '@mui/icons-material/ArrowBack';
-import Link from "next/link";
-import React, { useState } from 'react';
+import Link from "next/link"
+import React, {useState} from 'react';
 
-const Modeling = () => {
+const modeling = () => {
```

Comment:
Same here, Modeling. Also, the formatting on this page is very strange, make sure to run the formatter.